### PR TITLE
fix: Tests for Step 96-New Web design Certificate.

### LINKED ISSUE #49841: Step 96-New Web design Certificate
### Describe the Issue

Step 96Passed
Give wave four waypoints starting at 10%, and incrementing by 10%.

1.The tests allow duplicate waypoints for the four required percentages.
2.The tests allow other waypoint values not specified in the instructions.

### Affected Page

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-css-transforms-by-building-a-penguin/step-96

### Your code

```

```
@keyframes wave {
  10% {  

  }
  20% {

  }
  10% { 

  }
  30% {

  }
  10% {

  }
  40% {

  }
  10% {

  }
  50% {
    
  }
}

### Expected behavior

When I read the instructions, I thought it said do four wave points starting at ten and add ten to each one. Once I failed. I calculated it to do each ten and add ten percent to each one after. I guess even though I did not read the instructions the way they were adviced I found a bug. I passed and when I went to next step it automatically fixed my mistake and removed the 50% also.

### Screenshots

_No response_

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

### Additional context

The instructions from what I read confused me to do four tens and add ten percent under each one. I was able to find the bug due to this so yay I am pleased to help. By doing a total of four I was able to add 50 percent, on the last one too. My suggestion to fix the sentence on instructions, would be to not say make four waves starting at ten. Instead maybe you could say make four waves start at 10 percent and then for the rest add ten percent to each one for a total of four waves only.

### PR DESCRIPTION
**Pull request by ArslanYM**
- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #49841

### CODE REVIEW

**Comment by raisedadead** [comment]
As a new contributor, we encourage you to read our [contributing guidelines](https://contribute.freecodecamp.org).

We expect contributors to know about the process specific to this project. Following the guidelines religiously earns fellow maintainers' respect and saves everyone time.

<details>
<summary>Some examples of this are:</summary>

1. Do not edit files directly through GitHub – while you can, it's not a good idea.
2. Ensure you follow the PR checklist and not just tick things off; otherwise, we won't take you seriously.
3. Use the correct way to link issues in the description of the PR by updating the `XXXXXX`. Do not just add issue numbers everywhere and anywhere you feel like.
4. Keep mentions and review requests to a minimum. We understand you are excited about contributing, and our maintainers will get back to you when they can.
5. Do not work directly off your `main` branch - create a new branch for the changes you are working on.

</details>

We appreciate you taking the time to help us and hope to see more contributions from you.

Happy Contributing.

**Comment by ArslanYM** [comment]
@raisedadead Sorry for the mistake. Is the title suitable now? 
Are there any other issues I should be fixing.

File: curriculum/challenges/english/14-responsive-web-design-22/learn-css-transforms-by-building-a-penguin/619d324f5915c929f36ae91d.md [hunk 1]
```diff
@@ -35,6 +35,13 @@ You should add a `40%` waypoint for `@keyframes wave`.
 assert([...[...new __helpers.CSSHelp(document).getCSSRules('keyframes')].find(rule => rule?.name === 'wave')?.cssRules].some(css => css?.keyText === '40%'));
 ```
 
+You should have 4 waypoints in your keyframe.
+
+```js
+assert([...[...new __helpers.CSSHelp(document).getCSSRules('keyframes')].find(rule => rule?.name === 'wave')?.cssRules].length);
```
Comment (gikf) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-css-transforms-by-building-a-penguin/619d324f5915c929f36ae91d.md:41]:
```suggestion
assert([...[...new __helpers.CSSHelp(document).getCSSRules('keyframes')].find(rule => rule?.name === 'wave')?.cssRules].length === 4);
```

Comment (ArslanYM) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-css-transforms-by-building-a-penguin/619d324f5915c929f36ae91d.md:41]: I will be creating  another pr to fix this issue.

**Pull request closed without merging**